File: Jawa/Code/Hospital.java
import java.util.ArrayList;

public class Hospital implements ManageDoctors, ManagePatients {
    private String hospitalName;
    private ArrayList<Patient> patients;
    private ArrayList<Doctor> doctors;

    public Hospital(String name) {
        this.hospitalName = name;
        this.patients = new ArrayList<>();
        this.doctors = new ArrayList<>();
    }

    //get set
    public void setHospitalName(String name) {
        this.hospitalName = name;
    }

    public String getHospitalName() {
        return hospitalName;
    }

    //fungsi add pasien
    @Override
    public void addPatient(Patient p) {
        patients.add(p);
    }

    //fungsi add doctor
    @Override
    public void addDoctor(Doctor d) {
        doctors.add(d);
    }

    //fungsi print doctor dan pasien
    public void printInfo() {
        System.out.println("=>> RSUD " + hospitalName + " <<=");

        System.out.println("\nList Doctors:\n");
        for (Doctor d : doctors) {
            d.printDoctor();
        }

        System.out.println("\nList Patients:\n");
        for (Patient p : patients) {
            p.printPatient();
        }
    }
}

File: Uler/Code/Hospital.py
from Doctor import Doctor
from Patient import Patient

class Hospital:
    def __init__(self, name):
        self.hospital_name = name
        self.patients = []
        self.doctors = []

    #get set
    def set_hospital_name(self, name):
        self.hospital_name = name

    def get_hospital_name(self):
        return self.hospital_name

    #fungsi add pasien dan dokter
    def add_patient(self, patient: Patient):
        self.patients.append(patient)

    def add_doctor(self, doctor: Doctor):
        self.doctors.append(doctor)

    #fungsi print pasien dan dokter
    def print_info(self):
        print(f"\n=>> RSUD {self.hospital_name} <<=")

        print("\nList Doctors:\n")
        for d in self.doctors:
            d.print_doctor()

        print("\nList Patients:\n")
        for p in self.patients:
            p.print_patient()